### PR TITLE
Implementing the throttling pattern to maintain healthy systems with Java Spring Boot [Duplicate 6496]

### PR DESCRIPTION
**Pull request by afropolardev**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [ ] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #6660`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by afropolardev** [comment]
@hectorkambow I got someone review it for me and I looked at his work and worked from his system. I hope this passes

**Comment by afropolardev** [comment]
> ### Article checklist
> ## Attention
> ## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner.
> 
> **ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**
> 
> > As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.
> 
> #### Pre-submission checks
> 
> > Please ensure that you have only one open topic suggestion (issue) + in-review article (linked pull request) at a time. This will ensure that we complete the review process in a timely manner from inception to publishing
> 
> > If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.
> 
> > For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.
> 
> #### Formatting and structure checks
> 
>     * [x]  Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
> 
>     * [x]  Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
> 
>     * [x]  Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
> 
>     * [x]  Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
> 
>     * [x]  Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
> 
>     * [x]  Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
> 
>     * [x]  Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
> 
>     * [x]  Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
> 
>     * [ ]  Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
> 
> 
> #### Grammar & plagiarism checks
> 
>     * [x]  Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com)
> 
>     * [x]  Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
> 
>     * [x]  Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
> 
>     * [x]  Have you added sources for quotes and images that aren't yours?
> 
> 
> #### Technical checks
> 
>     * [x]  Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
> 
>     * [x]  Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
> 
>     * [x]  Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?
> 
> 
> #### Topic suggestion this closes
> 
> > Remove the backticks and add the issue number below to link and close your Topic Suggestion (issue) when your article has been published (PR has been merged). See this [video](https://youtu.be/TKJ4RdhyB5Y?t=278) for more details.
> 
> `This closes #issue-number`
> 
> > Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

**Comment by ahmadmardeni1** [comment]
Please @ your last peer reviewer and link your issue. @afropolardev

File: content/articles/implementing-the-throttling-pattern-to-maintain-healthy-systems-with-java-spring-boot/index.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+#### Introduction
+Throttling is a regular English word. But in programming and software engineering, it would carry a slightly different connotation. Basically, it makes sure that any client does not have uncontrollable or uncheckable access to given resources. By this, the resource is protected from abuse and mis-use.
+
+A simple but extremely useful explanation would be to picture a man and his young child in a restaurant. Upon arrival, a waiter comes along to take their order and suree enough, they place a few. The waiter quickly realises that this young child should not be allowed to indulge in too much food in little time and chooses not to serve after if the delay time has not passed. But for the man, a bit more room can be allowed for consumption.  In other words, throttling is a design pattern for controlling resource access.
```
Comment (hectorkambow) [inline content/articles/implementing-the-throttling-pattern-to-maintain-healthy-systems-with-java-spring-boot/index.md:4]: suree

**Comment by hectorkambow** [comment]
still noticing some minor errors - and overall the articles seems a bit light on substance. 

Will be closing to allow you more time to go through and polish up the article. @afropolardev

**Pull request closed without merging**